actionlist/app/actions/types.js

```diff
@@ -1,6 +1,8 @@
-export const TOGGLE_DRAWER = 'toggle_drawer';
+export const OPEN_DRAWER = 'open_drawer';
+export const CLOSE_DRAWER = 'close_drawer';
 export const FILTER_ACTION_LIST = 'filter_action_list';
 export const SORT_ACTION_LIST = 'sort_action_list';
 export const SELECT_DROPDOWN_OPTION = 'select_dropdown_option';
 export const RESET_FILTERS = 'reset_filters';
+export const DATE_FILTER_CHANGE = 'date_filter_change';
```

Comment:
You might consider "namespacing" these actions by making the type `'actionlist/filter/date_filter_change'` for clarity's sake and to avoid collisions in the future. The `types` file has also grown quite a bit, so you may consider breaking them out into separate files grouped by common usage.